src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -1917,7 +1917,7 @@ private SSHConnectionInfo()
             this.UserName = userName;
             this.ComputerName = computerName;
             this.KeyFilePath = keyFilePath;
-            this.Port = DefaultPort;
+            this.Port = DefaultPort; // TODO: don't override Port directive in ssh_config
```

Comment:
Rather than have TODO, perhaps fix https://github.com/PowerShell/PowerShell/issues/11432 in this PR

Reply:
I'll work on the fix for #11432.